go/cmd/remotetool/main.go

```diff
@@ -145,7 +147,12 @@ func main() {
 		}
 
 	case uploadDir:
-		if err := c.UploadDirectory(ctx, getPathFlag()); err != nil {
+		us, err := c.UploadDirectory(ctx, getPathFlag())
+		if *jsonOutput {
+			js, _ := json.MarshalIndent(us, "", "  ")
+			fmt.Printf("%s\n", js)
```

Comment:
Did you mean to write to `os.WriteFile(jsonOutput, js, 0o666)` ?

Reply:
I didn't, but sure, if that's what you prefer :-) Done.

Reply:
Sorry to reopen this, but wouldn't printing to stdout accommodate pipes and files (via redirection) while writing to a file complicates the pipe workflow?

Reply:
The precedent in the posix way of doing this is to specific "-" as output to stdout. I think we'd get the best of both worlds.

Reply:
Isn't that usually for tools that naturally expect files for in/out?

Reply:
Users want a tool to be composable. Since remotetool sends its status update to stderr and/or a file, I agree it is possible to use stdout to get the data. While in many case it's great, sometimes the user may want to not take a chance and use a named file instead. For example curl will output to stdout by default but may redirect to the user's choosing with `--output` and associated flags. Exceptions to that are use case where incrementally of the processing is needed so they do not offer the option to write to a file. A great example is `go test -json` where the user will want to process a list of json objects.

Reply:
To be clear, especially that I commented a lot here, I don't have a strong opinion and I'm fine with whatever @ola-rozenfeld or @mrahs prefers. If you prefer a bool `-json` flag it's okay too.

Reply:
Thanks for taking the time to bounce ideas! No strong opinions here either. `json=-` (which can be supported later) or `-json` works for me.

Ola, let me know if you're ready to merge and I'll hit the button.

Reply:
Okay, added support for "json=-" for stdout. That's it, should be ready to merge.
Thank you Anas and Marc Antoine for the quick reviews!